run-qa-checks

```diff
@@ -15,14 +15,10 @@ rm -rf ./openwisp_controller/geo/tests/.pytest_cache \
 jshint openwisp_controller/pki/static/admin/pki/js/*.js
 jshint openwisp_controller/config/static/config/js/*.js
 jshint openwisp_controller/connection/static/connection/js/*.js
-stylelint "openwisp_controller/config/static/config/css/*.css"
-stylelint "openwisp_controller/connection/static/connection/css/*.css"
-
-
-# CSS checks
-stylelint "openwisp_controller/connection/static/connection/css/*.css"
 
 openwisp-qa-check \
+  --csslinter
+  --jslinter
```

Comment:
you're forgetting the \ on both lines..

Reply:
Yes, I added these but then too some of the Jslint checks were failing, I will see and improve the PR.